soc/arm64/nxp_layerscape/ls1046a/soc.h

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2021 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef SOC_H
+#define SOC_H
+
+#include <sys/util.h>
+#include <toolchain.h>
+
+#ifndef _ASMLANGUAGE
+#include <devicetree.h>
+#endif /* _ASMLANGUAGE */
+
+#define UART_REG_ADDR_INTERVAL 1
```

Comment:
is this used anywhere?

Reply:
Yes, it is used by uart driver: drivers/serial/uart_ns16550.c

Jiafei.